profile/README.md

```diff
@@ -0,0 +1,93 @@
+<p align="center">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://storage.googleapis.com/passage-docs/github-md-assets/passage-by-1password-dark.png">
+      <source media="(prefers-color-scheme: light)" srcset="https://storage.googleapis.com/passage-docs/github-md-assets/passage-by-1password-light.png">
+      <img alt="Passage by 1Password Logo" src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-by-1password-light.png">
+    </picture>
+</p>
+
+[Passage by 1Password](https://1password.com/product/passage) unlocks the passwordless future with a simpler, more secure passkey authentication experience.
+Passage handles the complexities of [WebAuthn](https://docs.passage.id/home/webauthn), and allows you to implement passkeys with ease on web and native devices. 
+
+
+
+<table>
+  <tr>
+    <td valign="top">
+      <h3><img src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png"> Passkey <b>Flex</b></h3>
+      <a href="https://docs.passage.id/flex" target="_new">Add passkeys to an existing auth solution →</a>
+      <p>Upgrade your existing authentication flow to support passkeys. With Passkey Flex, you can choose how you incorporate passkeys: as an alternative to passwords at login and registration, as a multi-factor authentication method, or allow users to add a passkey to an existing account.</p>
+      <sub>Passkey Flex Libraries & SDKs</sub><br />
```

Comment:
Seems off that the heading is a subscript and the text below is larger. 


<img width="447" alt="Screenshot 2024-10-25 at 4 35 32 PM" src="https://github.com/user-attachments/assets/f575fbf7-5ebf-44f3-91f9-c21927cf18d4">

Reply:
+1

Reply:
They are actually the same size! Though the illusion is valid. I adjusted the font-weight and think it helps clarify the hierarchy a bit.

Reply:
I think this looks better if we don't want to add an actual semantic header in here! I'm fine with it.